translations/it/pack/return/rtptc.json

```diff
@@ -0,0 +1,89 @@
+[
+    {
+        "code": "52001",
+        "name": "Colt Calibro 32",
+        "text": "Utilizzo (6 munizioni).\n[action]Spendi 1 munizione: <b>Combatti</b>. Infliggi +1 danno in questo attacco.\n[free]Spendi 1 risorsa: Rimetti la Colt Calibro 32 nella tua mano.",
+        "traits": "Oggetto. Arma. Arma da Fuoco.",
+        "slot": "Mano"
+    },
+    {
+        "code": "52002",
+        "name": "\"Mangiati un Po' di Piombo!\"",
+        "text": "Veloce. Gioca solo quando attivi la capacità <b>Combatti</b> su un supporto [[Arma da Fuoco]].\nSpendi 1 munizione aggiuntiva da quel supporto. Quando riveli i segnalini caos in questo attacco, rivela 1 segnalino aggiuntivo. Scegline 1 da risolvere e ignora l’altro.",
+        "traits": "Tattica."
+    },
+    {
+        "code": "52003",
+        "flavor": "\"Sono il più saggio tra gli uomini viventi, poiché so solo una cosa, e cioè di non sapere niente.\"\n- Platone, <u>La Repubblica</u>",
+        "name": "Deduzione Logica",
+        "text": "Per ogni indizio che possiedi (massimo 3), puoi scegliere se curare 2 orrori a un investigatore nel tuo luogo oppure scartare 1 carta [[Terrore]] nel tuo luogo.",
+        "traits": "Intuizione."
+    },
+    {
+        "code": "52004",
+        "name": "Geroglifici Arcani",
+        "subname": "Marchi di Iside",
+        "text": "Puoi includere questo supporto nel tuo mazzo solo migliorandolo dai Geroglifici Arcani (<i>Non Tradotti</i>) e solo se \"i geroglifici sono stati tradotti\".\nUtilizzo (3 cariche).\n[action]Spendi 1 carica: <b>Indaga</b>. Se hai successo, puoi mettere in gioco 1 supporto dal costo stampato pari o inferiore a X dalla tua mano, dove X è l’ammontare di punti di cui hai avuto successo.",
+        "traits": "Incantesimo",
+        "slot": "Arcano"
+    },
+    {
+        "code": "52005",
+        "name": "Furtività",
+        "text": "[free]Durante il tuo turno, esaurisci la Furtività: <b>Eludi</b>. Il nemico scelto riceve -2 Agilità in questo tentativo di elusione. Se hai successo, disimpegnati dal nemico ma non esaurirlo. Fino alla fine del tuo turno, quel nemico non può impegnarti.",
+        "traits": "Talento."
+    },
+    {
+        "code": "52006",
+        "name": "Suggestione",
+        "text": "Utilizzo (3 cariche). Se non ci sono cariche sulla Suggestione, scartala.\n[action]Esaurisci la Suggestione: <b>Eludi</b>. Aggiungi il tuo valore di [willpower] al tuo valore di abilità in questo tentativo di elusione. Se non hai successo di 2 o più punti, rimuovi 1 carica dalla Suggestione.",
+        "traits": "Incantesimo.",
+        "slot": "Arcano"
+    },
+    {
+        "code": "52007"
```

Comment:
still missing one here :P 
If you look at the diff here on hithub you can see the errors that there are errors highlighted in red (usually means there is some invalid json just before it)

Reply:
Well, I'm blind, or dumb, or both xD
I'm always missing something, sorry about that